### ISSUE TITLE
Intro reading: Add rain/umbrella

### ISSUE THREAD

**Issue by nickeubank** [nickeubank/ds4humans#8]
Students don't quite grok the distinction between passive-prediction and causal. Add the rain example? Umbrella use (passively) predicts rain (if we see one, we don't expect the other), but there's no causal relationship (if I were to manipulate umbrella use, it wouldn't cause rain / we wouldn't "predict" rain to start).

**Comment by joshclinton** [comment]
Sounds good.  If you want to be more provocative we could also raise the issue of demographics and outcomes here?   Or note the connection to these bad AI algorithms -- e.g., Chicago predicting experiencing violence/crime as a function of race, but race is clearly not causal.  The umbrella is a good example, but I wonder if a more provocative example is also useful/important  for making the point that this distinction matters a lot and that people confuse these two foundational purposes.  Correlation != causation

**Comment by nickeubank** [comment]
Race is a whole thing to think about... I mean, it kinda _is_ causal in the "what is race but a bundle of treatments" sense—where you live, your economic opportunities, education, how police treat you, etc. Just not in the "skin color --> crime" sense.

**Comment by joshclinton** [comment]
Agreed.  Just thinking about the misinterpretation of race is related to crime therefore race causes crime.  (or poverty, or standardized test scores, or whatever)....   Precisely because we cannot measure all of those other confounded correlates of the outcome things get attributed to race that seem extremely unlikely.   Or gender, or anything.  Perhaps suggests the importance of theory as to what is going on, but highlights the issues with throwing predictors in a model without thinking what those correlations may or may not mean
